ConditionsConsumed/runCMSDrivers_mc2022.sh

```diff
@@ -0,0 +1,68 @@
+
+
+#6: HLT
+echo "------------------------------"
+echo "Running step-5/8: HLT"
+echo "------------------------------"
+#DIGI:pdigi_valid,L1,DIGI2RAW,HLT:@relval2022
+cmsDriver.py step6  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s HLT:@relval2022  --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step3_DIGI_L1_DIGI2RAW.root --fileout output_step6_HLT.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' --outputCommands "keep *" |& tee output_step6_HLT.log
+
+
+echo "------------------------------"
+echo "Running step-5.5/8: Digi-DIGI2RAW-L1-HLT"
+echo "------------------------------"
+#6.1: DIGI_L1+DIGI2RAW+HLT
+cmsDriver.py step3  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s DIGI:pdigi_valid,L1,DIGI2RAW,HLT:@relval2022 --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step2_SIM.root --fileout output_step3_DIGI_L1_DIGI2RAW_HLT.root  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' --outputCommands 'keep *_simCastorDigis_*_*' |& tee output_step3_DIGI_L1_DIGI2RAW_HLT.log
+
+
+#7 AODSIM (RAW2DIGI,L1Reco,RECO,RECOSIM,EI)
+echo "------------------------------"
+echo "Running step-6/8: AODSIM"
+echo "------------------------------"
+cmsDriver.py step7 --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s RAW2DIGI,L1Reco,RECO,RECOSIM --datatier AODSIM --eventcontent AODSIM --filein file:output_step3_DIGI_L1_DIGI2RAW_HLT.root --fileout output_step7_AODSIM.root --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' |& tee output_step7_AODSIM.log
+
+
+#7.1 AODSIM+DIGI_L1+DIGI2RAW+HLT
+
+#8 MINIAODSIM
+echo "------------------------------"
+echo "Running step-7/8: MINIADOSIM"
+echo "------------------------------"
+cmsDriver.py step8 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s PAT --datatier MINIAODSIM --eventcontent MINIAODSIM --filein file:output_step7_AODSIM.root --fileout output_step8_MINIAODSIM.root --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' |& tee output_step8_MINIAODSIM.log
+
+
+
+#9 NANOAODSIM
+echo "------------------------------"
+echo "Running step-8/8: NANOAODSIM"
+echo "------------------------------"
+cmsDriver.py step5 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s NANO --datatier NANOAODSIM --eventcontent NANOAODSIM --filein file:output_step8_MINIAODSIM.root --fileout output_step9_NANOAODSIM.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' |& tee output_step9_NANOAODSIM.log
```

Comment:
I think it is OK for now, but we really should try to understand why it is not possible to run every single step separately. @MataiasCost , please let's investigate it further although it is not a showstopper for now, imho.

Reply:
The AODSIM, GEN and SIM are the only that we cannot run separately, but AODSIM, in the case of 2018, don't run separately too. I can give another try in the case of GEN and SIM.

Source: https://github.com/ravindkv/MyAlCaTools/blob/master/runCMSDrivers_mc2018.sh

Reply:
No, that's fine. the GEN-SIM step is fine.